docs/docs/integration/embedding.md

```diff
@@ -89,21 +89,21 @@ The response will contain an iframe URL that can be used to embed the dashboard
 The frontend just need to use the iframeSrc to embed the dashboard in the customer's application. Here's an example of how to embed the dashboard using iframe:
 
 ```html
-<iframe src="<iframeSrc>" width="100%" height="1000" frameborder="0" ></iframe>
+<iframe title="rill-dashboard" src="<iframeSrc>" width="100%" height="1000" style="border:0;" />
 ```
 
 ### Example
 Here's an example of how to create a dashboard component in a React application:
 
 ```jsx
 import React, { useEffect, useState } from 'react';
-import { useParams } from 'react-router-dom';
 
-const Dashboard = () => {
+const RillDashboard = () => {
   const [iframeUrl, setIframeUrl] = useState('');
 
   useEffect(() => {
-    const getIframeUrl = async () => {
+      const { userEmail } = useUser();
```

Comment:
The `useUser()` hook should go at the top-level above `useEffect`. Also, it'd be good to show its import, like:
```jsx
import { useUser } from '../hooks/useUser';
```

Reply:
sure